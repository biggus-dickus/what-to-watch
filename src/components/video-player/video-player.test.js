import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player';


const props = {
  poster: `https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/e4/e4224a21287a6983fa0cc3ffcb310230c22e1538_full.jpg`,
  src: `/etc/na-zdorovnya.mp4`,
  isHovered: false,
  muted: true,
  width: `640`,
  height: `480`
};

const tree = renderer.create(<VideoPlayer {...props} />).toJSON();

it(`<VideoPlayer /> should render correctly`, () => expect(tree).toMatchSnapshot());
