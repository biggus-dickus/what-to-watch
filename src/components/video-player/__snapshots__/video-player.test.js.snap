// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer /> should render correctly 1`] = `
<video
  height="480"
  muted={true}
  poster="https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/e4/e4224a21287a6983fa0cc3ffcb310230c22e1538_full.jpg"
  src="/etc/na-zdorovnya.mp4"
  width="640"
/>
`;
